Source/Particles/ElementaryProcess/Ionization.H

```diff
@@ -124,7 +124,7 @@ struct IonizationFilterFunc
                                );
 
             // Compute probability of ionization p
-            amrex::Real w_dtau = 1._rt/ ga * m_adk_prefactor[ion_lev] *
+            amrex::Real w_dtau = (E == 0._rt) ? 0._rt : 1._rt/ ga * m_adk_prefactor[ion_lev] *
```

Comment:
One thought: potentially, one could also add via `<limits>`
```suggestion
            amrex::Real w_dtau = (E <= std::numeric_limits<amrex::Real>::min()) ? 0._rt : 1._rt/ ga * m_adk_prefactor[ion_lev] *
```
https://en.cppreference.com/w/cpp/types/numeric_limits/min

or any other epsilon that makes sure that the `std::exp()` does not explode in value range for typical `m_adk_exp_prefactor` values with small `E` values.

But we can cross that bridge when we come to it (in tests/CI :) ).